### DISCUSSION TITLE
Language displayed in placeholder #_EVENTPRINTBOOKINGSLINK

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#679] [General]
Hi,
I use the #_EVENTPRINTBOOKINGSLINK placeholder on a page to display the list of bookings. I've created an eme.css and eme-print.css file in my theme directory. The column are always in English, whatever the language of the site. I'd like the list to be displayed in French, is that possible?

**Comment by liedekef** [comment]
works fine here ... is your user language set correctly?

**Comment by f14m07** [comment]
Well, on my Local site have a look to this setting
<img width="850" alt="image" src="https://github.com/user-attachments/assets/b9dac385-2252-4dd3-9f41-335bcec286f7">
and the result
<img width="1831" alt="image" src="https://github.com/user-attachments/assets/a2fb626a-ee0e-4b70-8828-d62f22645ee2">
You can see Phone number Status Paid.
So I don't understand

**Reply by liedekef** [reply]
and what is your user language? Not the site language?

**Reply by liedekef** [reply]
and (but unsure): did you check with another theme? And is the translation correct in the WP/EME backend interface when clicking on the "booking report" link?

**Reply by liedekef** [reply]
ok, I figured it out: the link points to the admin pages, and there your user language is taken into account. So I assume in the backend your language is set to English, resulting in this to be English too.

**Comment by f14m07** [comment]
Ok Franky, a thousand apologies, I'd forgotten about that setting I'd made to make it easier to chat on github.
Thank you for your patient and persevering support.
Everything's OK now
